MainModule/Server/Commands/Fun.lua

```diff
@@ -1092,7 +1092,23 @@ return function(Vargs, env)
 				end
 			end
 		};
-
+		Tripmine = { --// D I S A S T E R
+			Prefix = Settings.Prefix;
+			Commands = {"tripmine", "subspacetripmine","subspace"};
+			Args = {"player"};
+			Description = "Gives the target player(s) a Subspace Tripmine";
+			AdminLevel = "Admins";
+			Fun = true;
+			Function = function(plr: Player, args: {string})
+				local subspace = service.Insert(11999247)
```

Comment:
Shouldn't this be deleted once the command is finished? I'm not sure if the GC deletes instances that are unused

Reply:
could be.